apps/docs/tests/support/pageObject/DetailsPageObject.ts

```diff
@@ -0,0 +1,12 @@
+export class DetailsPageObject {
+  get frameworkName() {
+    return cy.get('main').find('div[class="mb-8"]').first().find('span').first();
+  }
+  header(name: string) {
+    return cy.get(`h1[id="${name}"]`);
```

Comment:
```suggestion
    return cy.get(`h1[id="${name}"]`);
    return this;
```
it would be nice if we could chain methods from objects